group/edwards25519/scalar.go

```diff
@@ -136,15 +137,20 @@ func (s *scalar) Pick(rand cipher.Stream) kyber.Scalar {
 	return s.setInt(i)
 }
 
-// SetBytes s to b, interpreted as a little endian integer.
+// SetBytes s to b, interpreted as a big endian integer.
+// The scalar will then be reduced modulo the prime order.
 func (s *scalar) SetBytes(b []byte) kyber.Scalar {
-	return s.setInt(mod.NewIntBytes(b, primeOrder, mod.LittleEndian))
+	return s.setInt(mod.NewIntBytes(b, primeOrder, mod.BigEndian))
```

Comment:
And to be able to save a useless allocation, we can use `scReduce` (removed in this PR) instead of allocation a big.Int and making a reverse copy in LE of the buffer (just this line).

Reply:
I wondered what scReduce was.